### ISSUE TITLE
versionComparable missing after restoring a revision of a migrated API

### ISSUE THREAD

**Issue by RakhithaRR** [wso2/api-manager#142]
### Description

The `versionComparable` parameter in registry artifact goes missing when a migrated API is restored to a previous revision.

### Steps to Reproduce

- Migrate an API from an earlier version (3.2.0)
- Go to the management console. The `versionComparable` value can be seen in the registry artifact.
- Go the Publisher and restore that API to a previous revision
- After going back to the management console, the value will be missing from the artifact

### Affected Component

APIM

### Version

4.1.0

**Comment by dushaniw** [comment]
@RakhithaRR did we fix this, or is the issue still persists in the migration?

**Comment by RakhithaRR** [comment]
No, we decided to skip this at that time. We need to check and see whether this is something that need to be fixed from product side or migration client side.
